images/win/scripts/Installers/Install-PowershellCore.ps1

```diff
@@ -2,5 +2,4 @@
 ##  File:  Install-PowershellCore.ps1
 ##  Desc:  Install PowerShell Core
 ################################################################################
-
-Invoke-Expression "& { $(Invoke-RestMethod https://aka.ms/install-powershell.ps1) } -UseMSI -Quiet"
+Install-Choco "powershell-core"
```

Comment:
What is the issue with installation from `https://aka.ms/install-powershell.ps1`. I didn't notice any issues with it previously.

Reply:
I have looked at it deeper, it looks like both [install-powershell.ps1](https://raw.githubusercontent.com/PowerShell/PowerShell/master/tools/install-powershell.ps1) and [choco install powershell-core](https://gitlab.com/DarwinJS/ChocoPackages/-/blob/d57372d292b4166313363fa262460bccb2ad29e9/pwshpackages/powershell-core/tools/chocolateyinstall.ps1) do the same things: download installer from `https://github.com/PowerShell/PowerShell/releases/download/v${release}/${packageName}` and apply it.
The single difference: `install-powershell.ps1` determines latest version in [runtime](https://raw.githubusercontent.com/PowerShell/PowerShell/master/tools/metadata.json) but in `choco install powershell-core` the version is [hardcoded](https://gitlab.com/DarwinJS/ChocoPackages/-/commit/d57372d292b4166313363fa262460bccb2ad29e9) and package owner has to update it manually every time when the new version is released.
Considering this point, I would prefer to continue using `https://aka.ms/install-powershell.ps1`.
@alepauly for any thoughts

Reply:
reverted